docs/learn/bitesize/bitesize-dynamicprefabs.md

```diff
@@ -24,7 +24,7 @@ There's also the **APIPlayground**, which serves as an API playground that imple
 
 ### Scene 00_Preloading Dynamic Prefabs
 
-The `00_Preloading Dynamic Prefabs` scene is the simplest implementation of a dynamic Prefab. It instructs all game instances to load a network Prefab (it can be just one, it can also be a set of network Prefabs) and inject them to NetworkManager's NetworkPrefabs list before starting the server. What's important is that it doesn't matter where the Prefab comes from. It can be a simple Prefab or it can be an Addressable - it's all the same.
+The `00_Preloading Dynamic Prefabs` scene is the simplest implementation of a dynamic Prefab. It instructs all game instances to load a network Prefab (it can be just one, it can also be a set of network Prefabs) and inject them to a ScriptableObject's NetworkPrefabs list before starting the server. What's important is that it doesn't matter where the Prefab comes from. It can be a simple Prefab or it can be an Addressable - it's all the same.
```

Comment:
This whole file should remain as it was. The workflow and API for dynamic prefabs is still done through NetworkManager.